app/utils/auth0_parser.rb

```diff
@@ -0,0 +1,22 @@
+class Auth0Parser
+  def authenticate_request(request)
+    @request = request
```

Comment:
Instance variables would create a problem.
How many instances of the parser exist in the app (when running the server)?
What happens if multiple requests are being processed in parallel?